website/docs/pipelines/promoting-applications.mdx

```diff
@@ -130,133 +133,186 @@ Be sure to create the Provider/Alert tuple on **each of the leaf clusters
+roles created before policy agent was introduced as admission controller.
 
-```shell
-# example to use git over https with basic auth and pat
-$ kubectl create secret generic promotion-credentials \
-  --namespace=pipeline-01 \
-  --from-literal="username=<bot account name>" \
-  --from-literal="password=<token value>" \
-  --from-literal="token=<token value>" \
-  --from-literal="hmac-key=<hmac-key value>"
-```
+Once you have enabled Policy, the [Policy Library](../../policy/weave-policy-profile/) gives you a set of
+good practices policies that will help you securing your pipeline secrets according to previous RBAC recommendations. Deploy
+them as Kustomization based on the following example:
 
 ```yaml
+apiVersion: source.toolkit.fluxcd.io/v1
+kind: GitRepository
+metadata:
+  name: policy-library
+  namespace: flux-system
+spec:
+  interval: 10m0s
+  url: https://github.com/weaveworks/policy-library.git
+  secretRef:
+    name: policy-library-github-credentials
 ---
-apiVersion: v1
-kind: Secret
+apiVersion: kustomize.toolkit.fluxcd.io/v1
+kind: Kustomization
 metadata:
-  name: promotion-credentials
-  namespace: pipeline-01
-data:
-  username: ZXhhbXBsZQ==
-  password: ZXhhbXBsZS1wYXNzd29yZA==
-  token: Z2hwX01IL3RsTFpXTXZMY0FxVWRYY1ZGL0lGbzh0WDdHNjdsZmRxWQ==
-  hmac-key: OEIzMTNBNjQ0REU0OEVGODgxMTJCQ0VFNTQ3NkE=
-type: Opaque
+  name: rbac-secrets-good-practices
+  namespace: flux-system
+spec:
+  interval: 1m0s
+  sourceRef:
+    kind: GitRepository
+    name: policy-library
+  path: ./goodpractices/kubernetes/rbac/secrets
+  prune: true
 ```
 
-:::tip
-- The Git provider token provided in the `token` field needs to be given permission to create pull requests in the pipeline's repository (defined in `.spec.promotion.strategy.pull-request.url`).
-- The `hmac-key` field must match the key used for the Provider resource (.spec.secretRef), if specified in the leaf clusters.
-:::
-
-</details>
+:::caution
 
-### Define promotion in pipeline resource
+Policies might have exclusion features to accommodate privileged workloads like Flux. You could manage them via [PolicyConfig](https://docs.gitops.weave.works/docs/policy/policy-configuration/).
+For example, in order to allow Flux you could use the following `PolicyConfig`:
 
-The field `.spec.promotion.strategy.pull-request` defines details about the Git repository used for promoting the given app.
-Set the `secretRef.name` field to the name of the Secret created in the previous step and the `url` and `branch` fields to the
-Git repository's HTTPS URL and optionally a specific branch (if the branch is not set, it defaults to `main`).
-If using the `generic-hmac` Provider from leaf clusters, also set the `.spec.promotion.strategy.secretRef.name` to the name of the Secret created previously.
+```yaml
+apiVersion: pac.weave.works/v2beta2
+kind: PolicyConfig
+metadata:
+  name: allow-flux
+spec:
+  match:
+    apps:
```

Comment:
This example shows how you could leverage PolicyConfig to add exclusions for privilege workloads for your policies. 

In our example, we are deploying the goodpractices kustomization that deploy policies like `weave.templates.rbac-prohibit-wildcards-policyrule-resources`. These policies by default does not have exclusions. In case you want to have exclusions (see my comment [here](https://github.com/weaveworks/policy-library/pull/31/files#r1273143158)) you leverage PolicyConfig.